pkg/pillar/go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/prometheus/procfs v0.7.3
```

Comment:
There is a extra space in your commit message (I've marked both spaces with XX):
```
"gostats will make the udp connection by itself,XXthere is no need for
g2s package anymore."
```